### DISCUSSION TITLE
Errors in view-general.ts: Cannot find module 'lit/decorators.js' or its corresponding type declarations.ts(2307)

### DISCUSSION THREAD

**Discussion by Hohenloherin** [jeroenterheerdt/HAsmartirrigation#507] [General]
What am I doing wrong to get these errors (7 in total)? I have just installed the module, but cannot find smart irrigation panel. so I looked in the config and found a red dot indicating errors. 
![errors-in-view-general](https://github.com/user-attachments/assets/3d4bdf74-f911-46cd-a2f0-7eab84cdb605)
Any help is very much appreciated. Thanks in advance.

**Comment by jeroenterheerdt** [comment]
question: what are you doing? Are you installing this in HA? Manually? Through HACS?

**Reply by Hohenloherin** [reply]
I am running HA on a rpi4b with OS. I installed smart irrigation via HACS, got an api-key from owm and started to integrate smart irrigation into HA. The integration process is interrupted with an error: "Weather service API key incorrect". Checked key and everything, but could not succeed. Then I opened VSC and found a red dot, followed it to the above screenshot.
What is wrong?

**Reply by jeroenterheerdt** [reply]
thanks - how long before did you activate your API key? [As the docs says it can take a long time before the OWM key is actually active](https://jeroenterheerdt.github.io/HAsmartirrigation/installation-weatherservice.html#getting-open-weather-map-api-key). To rule out activation issues, do a request using your API key using Postman or equivalent...

**Reply by jeroenterheerdt** [reply]
also these `lit/decorators.js` issues are likely just a red herring.
